src/WebJobs.Extensions.DurableTask.Analyzers/Resources.resx

```diff
@@ -130,10 +130,10 @@
     <value>Activity function named '{0}' doesn't have an input. Pass null for the input parameter instead.</value>
   </data>
   <data name="ActivityArgumentAnalyzerTitle" xml:space="preserve">
-    <value>Activity function call is using the wrong argument type.</value>
+    <value>Activity function call is using the wrong argument type</value>
   </data>
   <data name="ActivityNameAnalyzerCloseMessageFormat" xml:space="preserve">
-    <value>Activity function named '{0}' does not exist. Did you mean '{1}'?</value>
+    <value>Activity function named '{0}' does not exist. You may have intended '{1}'.</value>
```

Comment:
What's the reasoning behind this change? I feel like "did you mean x?" is commonly used in other contexts?

Reply:
It is... but the compiler warns that MessageFormat strings should never end with a ?. Happy to evaluate other formulations of this string if we can find a better one.